.github/workflows/call-build-pkgdown.yml

```diff
@@ -0,0 +1,12 @@
+# Checks that the pkgdown site builds for a repository.
+# this assumes pkgdown is already set up.
+name: call-build-pkgdown
+# on specifies the build triggers. See more info at https://docs.github.com/en/actions/learn-github-actions/events-that-trigger-workflows
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
```

Comment:
Just wanted to check: Do we want to only run this on the main branch? I could also see this being helpful for pull requests to main, for instance.

Reply:
I was thinking on all PR and a dispatch mode?

Reply:
@kellijohnson-NOAA, just to double check, you mean all PRs to main, right? And remove running on push to main?

If so, I think we can modify the build triggers to look like
```yml
on:
  workflow_dispatch:
  pull_request:
    branches:
      - main
```

Reply:
Yes, that is what I meant because we already have the build and deploy on pushes to main.

Reply:
Thank you, @k-doering-NOAA and @kellijohnson-NOAA for catching this. The build trigger has been updated.